src/models.rs

```diff
@@ -3,10 +3,25 @@ use sqlx::FromRow;
 
 #[derive(Debug, Serialize, Deserialize, FromRow)]
 pub struct RedeemableSubscription {
-    pub sub_id: String,
-    pub module: String,
+    pub id: String,
     pub subscriber: String,
     pub recipient: String,
     pub amount: String,
+    pub category: Category,
     pub next_redeem_at: i32,
 }
+
+#[derive(Debug, Serialize, Deserialize, sqlx::Type)]
+#[sqlx(type_name = "integer", rename_all = "lowercase")]
+#[serde(rename_all = "lowercase")]
+pub enum Category {
+    #[sqlx(rename = "0")]
+    #[serde(rename = "trusted")]
+    Trusted = 0,
+    #[sqlx(rename = "1")]
+    #[serde(rename = "untrusted")]
+    Untrusted = 1,
+    #[sqlx(rename = "2")]
+    #[serde(rename = "group")]
+    Group = 2,
+}
```

Comment:
Still need to test this.

Reply:
Sample Response:

[
  {
    "id": "0xf7562b10c4c05105f7c577196d1699f6d2fc77a1b95e4f6ccd69fcf9543ed6df",
    "subscriber": "0xcf6dc192dc292d5f2789da2db02d6dd4f41f4214",
    "recipient": "0xc175a0c71f1eda836ebbf3ab0e32fc8865fdee91",
    "amount": "10000000000000000",
    "category": "trusted",
    "next_redeem_at": 0
  }
]